lib/grunt/utils.js

```diff
@@ -76,9 +76,9 @@ module.exports = {
 
 
   wrap: function(src, name) {
-    src.unshift('src/' + name + '.prefix');
-    src.push('src/' + name + '.suffix');
-    return src;
+    var files = ['src/' + name + '.prefix', 'src/' + name + '.suffix'];
+    files.splice(1, 0, src);
+    return files;
```

Comment:
This would be more readable:
```js
wrap: function(src, name) {
  return [`src/${name}.prefix`, ...src, `src/${name}.prefix`];
},
```
or even:
```js
wrap:(src, name) => [`src/${name}.prefix`, ...src, `src/${name}.prefix`],
```
Let's not be afraid of ES6+ in Node scripts. :)

Reply:
I did think about that but I thought we were generally sticking with ES5 for AngularJS code.

Reply:
I went with:

```
  wrap(src, name) {
    return [`src/${name}.prefix`, ...src, `src/${name}.prefix`];
  },
```

Reply:
Doh! not prefix prefix!!

Reply:
We have ESLint configured to allow ES2017 for Node scripts and we already use ES6 in Firebase code. :) It's true `Gruntfile.js` is mostly ES5 for now, though.